runtime/polkadot/Cargo.toml

```diff
@@ -18,6 +18,7 @@ babe-primitives = { package = "sp-consensus-babe", git = "https://github.com/par
 sp-api = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 inherents = { package = "sp-inherents", git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 offchain-primitives = { package = "sp-offchain", git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
+sp-application-crypto = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
```

Comment:
Same as above, needs `sp-application-crypto/std` below.

Reply:
Not sure if needed actually.

Reply:
yep, it's not needed here, and in `common` it is only needed as a dev dep, thanks